src/xo-app/vm/tab-console.js

```diff
@@ -20,21 +20,20 @@ export default class TabConsole extends Component {
     this.refs.noVnc.sendCtrlAltDel()
   }
 
-  _storeVmClipboard = clipboard => {
+  _getRemoteClipboard = clipboard => {
     this.setState({ clipboard })
     this.refs.clipboard.value = clipboard
   }
-  _setClipboard = event => {
-    const { value } = event.target
-    debounce(value => {
-      this.setState({ clipboard: value })
-      this.refs.noVnc.setClipboard(value)
-    }, 200)(value)
+  __setRemoteClipboard = debounce(value => {
+    this.setState({ clipboard: value })
+    this.refs.noVnc.setClipboard(value)
+  }, 200)
+  _setRemoteClipboard = event => {
```

Comment:
You can use `invoke` to create a intermediary function without polluting the current object.